src/status_im/ui/screens/wallet/styles.cljs

```diff
@@ -43,10 +48,10 @@
 
 (defn button-container [enabled?]
   (merge
-    {:flex-direction :row
-     :align-items    :center}
-    (when-not enabled?
-      {:opacity 0.4})))
+   {:flex-direction :row
```

Comment:
again, why does this happen?

Reply:
I think it's because a function name alone on a line and its arguments coming after is not very idiomatic so emacs doesn't deal with it

Reply:
https://github.com/bbatsov/clojure-style-guide#one-space-indent